airbyte-integrations/connectors/source-harvest/unit_tests/unit_test.py

```diff
@@ -1,51 +0,0 @@
-#
```

Comment:
I feel like we should keep those tests. What was the reason for removing them? Maybe `test_next_page_token` needs more work and therefore I'm whiling to drop it but the rest should be pretty straighforward

Reply:
@maxi297 

I guess I'm a bit confused why these tests would be necessary when this functionality is effectively already tested by the low code CDK unit tests as well as the CATs -- `invoices`, `invoice_payments`, and `expenses_clients` all return records for the sandbox account. Let me know your thoughts.

Reply:
Added the tests back and updated to function with the new low-code implementation.